### DISCUSSION TITLE
how to convert a multiple jpeg images into one tiff image

### DISCUSSION THREAD

**Discussion by shameerasl** [ImageMagick/ImageMagick#4977] [Development]
i wanted to convert multiple jpg images into one tiff image.is there is any code or references available. i am sorry if i am blundering. i am just a beginner in this.thanks in advance

**Comment by fmw42** [comment]
If on ImageMagick 6, 

`convert image1.suffix image2.suffix ....imageN.suffix image.tiff`

If on ImageMagick 7, change "convert" to "magick"

This will make a multi-page tiff image.

For new users, see


http://www.imagemagick.org/discourse-server/viewtopic.php?f=1&t=9620
http://www.imagemagick.org/script/command-line-processing.php
https://imagemagick.org/script/command-line-options.php
http://www.imagemagick.org/Usage/reference.html
http://www.imagemagick.org/Usage/
https://github.com/ImageMagick/usage-markdown
https://imagemagick.org/script/porting.php#cli
https://legacy.imagemagick.org/discourse-server/index.php

**Reply by shameerasl** [reply]
Thanks for your reply. it worked fine.
i have another query like how to reduce the size of a tiff file(multipage tiff file) I am using ImageMagick 7. thanks in advance.

**Reply by fmw42** [reply]
You can compress the tiff file using -compress. See https://imagemagick.org/script/command-line-options.php#compress  Typically, LZW or ZIP or JPG for color or grayscale and Group4 for binary.

**Reply by shameerasl** [reply]
i have around 700 tiff images and and each has around 25 mb size. i want to reduce images size to 2 mb. and combine to a tiff file..... hope you can help me... any idea or references  will be great help....

**Reply by fmw42** [reply]
I do not think you can reduce the file size from 25 mb to 2 mb with just LZW or ZIP compression, but possibly JPG compression might work. However, the quality might not be to your liking.  Alternately, you would have to reduce the dimensions.
